controllers/licenseissuer/internal/controller/license_webhook.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2023.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	cloud "github.com/labring/sealos/controllers/licenseissuer/internal/manager"
```

Comment:
package name.

Reply:
？？？if no package name， can i pass the ci ?

Reply:
rename `cloud`?